MediaBrowser.Model/Lyrics/LyricLineTimeTag.cs

```diff
@@ -0,0 +1,28 @@
+namespace MediaBrowser.Model.Lyrics;
+
+/// <summary>
+/// LyricLineTimeTag model.
```

Comment:
Not really descriptive. Should at least give some idea of what this is used for.